lib/events/README.md

```diff
@@ -73,6 +74,7 @@ If all values will be the same -> event_id will be the same -> event will be upd
 * `event_id_seed_optional` - string, optional part used for event_id generation. By default - ""(empty string)
 * `occurred_at` - optional, datetime, if provided - overwrite `current datetime` sent by default
 * `version` - optional, string, overwrite default "1.0.0" value
+* `event_json_schema` - JSON schema to validate event.
```

Comment:
it will be good to add some instructions on how to build a schema

Reply:
Do you think the instructions should be here? In the end, defining a JSON schema is not this service responsibility. This service should get a valid JSON schema.

And what kind of instruction would you add? In the end is a matter of building a JSON schema for your event struct, which is a standard. Not sure what we should add.